packages/suite/src/middlewares/wallet/__tests__/coinmarketMiddleware.test.ts

```diff
@@ -11,6 +11,11 @@ import suiteReducer from 'src/reducers/suite/suiteReducer';
 jest.mock('src/services/suite/invityAPI');
 invityAPI.setInvityServersEnvironment = () => {};
 invityAPI.createInvityAPIKey = () => {};
+invityAPI.getInfo = async () =>
+    await {
+        coins: {},
+        platforms: {},
+    };
```

Comment:
No need of async/await
```suggestion
invityAPI.getInfo = () =>
    Promise.resolve({
        coins: {},
        platforms: {},
    });
```